scripts/Install-VSCode.ps1

```diff
@@ -132,7 +134,7 @@ param(
     [string]$Architecture = "64-bit",
 
     [parameter()]
-    [ValidateSet("Stable", "Insider-System", "Insider-User")]
+    [ValidateSet("Stable","Stable-User", "Insider-System", "Insider-User")]
```

Comment:
I know you didn't do it, but it bothers me that these are double quotes yet the same ValidateSet on line 204 is single quotes...

And a nit: space between comma and "Stable-User"

Reply:
Most of the script uses single quotes. I could change the double quotes when suitable.

Reply:
Done in 2db94cf82c6a20127cf980f85f7d1d52bb346ca6